docs/components/menus.md

```diff
@@ -0,0 +1,353 @@
+
+### Cascading menu indicator attributes
+
+&nbsp;         | Property
+-------------- | ------------------------
+Adding cascading menu | Not currently supported.
+
+### Selection state attributes
+
+&nbsp;         | Property
+-------------- | ------------------------
+ | Use the `PopupMenuButton` widget. Within this widget use `itemBuilder`. Within the list of itemBuilder add a `CheckedPopupMenuItem`.
+
+## Types
+
+Menus allow users to make a selection from multiple options. They are less prominent and take up less space than selection controls, such as a set of radio buttons.
+
+There are two types of menus: 1\. [dropdown menus](#dropdown-menus), 2\. [exposed dropdown menus](#exposed-dropdown-menus).
+
+![Composite image of example dropdown and exposed dropdown menu types](assets/menus/menu_types_composite.png)
+
+### Dropdown menus
+
+Dropdown menus display a list of options, triggered by an icon, button, or action. Their placement varies based on the element that opens them.
+
+#### Dropdown menu example
+
+The following is an example of a dropdown menu:
+
+- [Class definition](https://api.flutter.dev/flutter/material/PopupMenuButton-class.html)
+- [GitHub source](https://github.com/flutter/flutter/blob/master/packages/flutter/lib/src/material/popup_menu.dart)
+- [Dartpad Demo](https://dartpad.dev/embed-flutter.html?gh_owner=material-components&gh_repo=material-components-flutter&gh_path=docs/components/dartpad/menus/dropdown&gh_ref=develop)
+
+![Dropdown menu example](assets/menus/dropdown_menu_example.png)
+
+```dart
+PopupMenuButton(
+  icon: Icon(Icons.more_vert),
+  itemBuilder: (BuildContext context) => <PopupMenuEntry>[
+    const PopupMenuItem(
+      child: ListTile(
+        leading: Icon(Icons.add),
+        title: Text('Item 1'),
+      ),
+    ),
+    const PopupMenuItem(
+      child: ListTile(
+        leading: Icon(Icons.anchor),
+        title: Text('Item 2'),
+      ),
+    ),
+    const PopupMenuItem(
+      child: ListTile(
+        leading: Icon(Icons.article),
+        title: Text('Item 3'),
+      ),
+    ),
+    const PopupMenuDivider(),
+    const PopupMenuItem(child: Text('Item A')),
+    const PopupMenuItem(child: Text('Item B')),
+  ],
+),
+```
+
+### Exposed dropdown menus
+
+Exposed dropdown menus display the currently selected menu item above the list of options. Some variations can accept user-entered input.
+
+#### Exposed dropdown menu example
+
+The following is an example of a dropdown menu:
+
+- [Class definition](https://api.flutter.dev/flutter/material/DropdownButton-class.html)
+- [GitHub source](https://github.com/flutter/flutter/blob/master/packages/flutter/lib/src/material/dropdown.dart)
+- [Dartpad Demo](https://dartpad.dev/embed-flutter.html?gh_owner=material-components&gh_repo=material-components-flutter&gh_path=docs/components/dartpad/menus/exposed_dropdown&gh_ref=develop)
+
+![Exposed Dropdown menu example](assets/menus/exposed_dropdown_menu_example.png)
```

Comment:
Please update alt-text to:

"Exposed dropdown menu example listing 6 options"

Reply:
Done